src/api/admin.ts

```diff
@@ -1,12 +1,17 @@
 import express from 'express';
-import { env } from '../env';
+import { Config } from '../entities/config';
 
 export const admin = express.Router();
 
 admin.use(express.json());
 
-admin.post('/login', (req, res): void => {
-  if (req.body.secret !== env.adminSecret) {
+admin.post('/login', async (req, res): Promise <void> => {
+  const adminSecret = await Config.findOneOrFail('adminSecret');
```

Comment:
Without a try/catch here, the app would become unusable in this state. For now, let's just use `findOne` and we can deal with the other half of this flow later